accounts/views_profile.py

```diff
@@ -65,4 +67,95 @@ def profile_update(request):
         else:
             form = ProfileForm(profile.get_populated_fields())
     template_data['form'] = form
-    return render(request, 'ienitk/profile/update.html', template_data)
\ No newline at end of file
+    return render(request, 'ienitk/profile/update.html', template_data)
+
+
+def apply(request):
+    authentication_result = views.authentication_check(request)
```

Comment:
Only the user with candidate role should be able to access this method. Please add necessary permission

Reply:
@chaitany10 
The correct way to add permission is 
```python
authentication_result = views.authentication_check(request, [Account.ACCOUNT_ADMIN, Account.ACCOUNT_CANDIDATE])
```
The second parameter is the array to whom we want to allow access to the method.
Do not put any if, else conditions to check.